Create a tutorial for setting Minio to the GCP Gateway mode (#3377)
kyma-project/kyma: docs/asset-store/docs/03-03-minio-gateway.md

The whole concept of the Asset Store relies on Minio as the storage solution. It supports Kyma's manifesto and the "batteries included" rule by providing you with this on-premise solution by default.

> For security reasons Minio credentials for object storage and web access are stored inside the Kubernetes `Secret` object. To access the credentials, run the following command:

Reviewer: ```suggestion
For security reasons, Minio credentials are generated during Kyma installation and stored inside the Kubernetes `Secret` object. To access the credentials, run the following command:
```

 ```bash

> kubectl get secret assetstore-minio -n kyma-system -o jsonpath=“{.data.accesskey}” | base64 -D

Reviewer: ```suggestion
#Access Key to Minio
kubectl get secret assetstore-minio -n kyma-system -o jsonpath=“{.data.accesskey}” | base64 -D
```

> kubectl get secret assetstore-minio -n kyma-system -o jsonpath=“{.data.secretkey}” | base64 -D

Reviewer: ```suggestion
#Secret Key to Minio
kubectl get secret assetstore-minio -n kyma-system -o jsonpath=“{.data.secretkey}” | base64 -D
```
